.github/workflows/windows-qt6.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         uses: hendrikmuhs/ccache-action@v1.2
         with:
           max-size: 1G
-
+          key: win-64-qt6-ccache-${{ github.base_ref }}
```

Comment:
I would suggest to avoid this as this creates way too much data volume and basically prevent us from having the cache we actually need.

In other workflow, I am only saving the cache when it's on master or a release* branch. I think this is a good compromise.

Reply:
@3nids I think since https://github.com/qgis/QGIS/pull/56856 there is a cache per run saved (which is quite heavy)

Reply:
yes you are forced to do this because you cannot have the same id twice (sadly). But again, we do this for the main branches only.

Reply:
it looks like hendrikmuhs/ccache-action@v1.2 doesn't support restore-keys which will be problematic here

Reply:
See https://github.com/hendrikmuhs/ccache-action/blob/main/action.yml

It does support the restore-key -- and would have a `save` option which could be disabled in pull requests